backend/tests/Designer.Tests/Controllers/OptionsController/GetTests.cs

```diff
@@ -0,0 +1,114 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Models;
+using Designer.Tests.Controllers.ApiTests;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Designer.Tests.Controllers.OptionsController;
+
+public class GetTests : DisagnerEndpointsTestsBase<GetTests>, IClassFixture<WebApplicationFactory<Program>>
+{
+    public GetTests(WebApplicationFactory<Program> factory) : base(factory)
+    {
+    }
+
+    [Fact]
+    public async Task GetOptionListIds_Returns_200OK_With_OptionListIds()
+    {
+        // Arrange
+        const string repo = "app-with-options";
+        string apiUrl = $"/designer/api/ttd/{repo}/options";
+        using HttpRequestMessage httpRequestMessage = new(HttpMethod.Get, apiUrl);
+
+        // Act
+        using HttpResponseMessage response = await HttpClient.SendAsync(httpRequestMessage);
+        string responseBody = await response.Content.ReadAsStringAsync();
+        string[] responseList = JsonSerializer.Deserialize<string[]>(responseBody);
+
+        // Assert
+        Assert.Equal(StatusCodes.Status200OK, (int)response.StatusCode);
+        Assert.Equal(2, responseList.Length);
+    }
+
+    [Fact]
+    public async Task GetOptionListIds_Returns_200OK_With_Empty_OptionListId_Array()
+    {
+        // Arrange
+        const string repo = "empty-app";
+        string apiUrl = $"/designer/api/ttd/{repo}/options";
+        using HttpRequestMessage httpRequestMessage = new(HttpMethod.Get, apiUrl);
+
+        // Act
+        using HttpResponseMessage response = await HttpClient.SendAsync(httpRequestMessage);
+        string responseBody = await response.Content.ReadAsStringAsync();
+        string[] responseList = JsonSerializer.Deserialize<string[]>(responseBody);
+
+        // Assert
+        Assert.Equal(StatusCodes.Status200OK, (int)response.StatusCode);
+        Assert.Empty(responseList);
+    }
+
+    [Fact]
+    public async Task GetSingleOptionList_Returns_200Ok_With_OptionList()
+    {
+        // Arrange
+        const string repo = "app-with-options";
+        const string optionListId = "test-options";
+
+        // This  option list matches the options in 'app-with-options'
+        var expectedOptionList = new List<Option>
+        {
+            new Option
+            {
+                Label = "label1",
+                Value = "value1",
+            },
+            new Option
+            {
+                Label = "label2",
+                Value = "value2",
+            }
```

Comment:
A solution can be to fecth the actual file and assert the file contents?

Reply:
Like this? 
````
    [Fact]
    public async Task GetSingleOptionsList_Returns200Ok_WithOptionsList()
    {
        // Arrange
        const string repo = "app-with-options";
        const string optionsListId = "test-options";

        string apiUrl = $"/designer/api/ttd/{repo}/options/{optionsListId}";
        using HttpRequestMessage httpRequestMessage = new(HttpMethod.Get, apiUrl);

        // Act
        using HttpResponseMessage response = await HttpClient.SendAsync(httpRequestMessage);
        string responseBody = await response.Content.ReadAsStringAsync();
        var responseList = JsonSerializer.Deserialize<List<Option>>(responseBody);

        // Assert
        Assert.Equal(StatusCodes.Status200OK, (int)response.StatusCode);
        Assert.Equal(2, responseList.Count);

        Assert.Equal("label1", responseList[0].Label);
        Assert.Equal("value1", responseList[0].Value);

        Assert.Equal("label2", responseList[1].Label);
        Assert.Equal("value2", responseList[1].Value);
    }
````

Reply:
I dont remember what I was thinking about in this case, but I still believe that the change you made here looks cleaner! 😅 👏